domains/acey.json

```
@@ -1,12 +1,9 @@
 {
-    "repo": "https://github.com/AceyDoCodes",
-    "description": "Email ;-;",
     "owner": {
         "username": "AceyDoCodes",
         "email": "aceygoesboom@gmail.com"
     },
     "record": {
-        "MX": ["mx1.improvmx.com", "mx2.improvmx.com"],
-        "TXT": "v=spf1 include:spf.improvmx.com ~all"
+        "TXT": "tbh i just need the other subdomain"
```

We don't support domain reserving, so please add a redirect to something.
Perhaps you would like to redirect it to your GitHub profile?
```suggestion
        "URL": "https://github.com/AceyDoCodes"
```